modules/mux/targets/host/test/UnitCL/host_clGetDeviceInfo.cpp

```diff
@@ -55,6 +55,10 @@ TEST_F(host_clGetDeviceInfoTest, Name) {
   const char *arch = "x86";
 #elif defined(__x86_64__) || defined(_M_X64)
   const char *arch = "x86_64";
+#elif defined(__riscv) && __riscv_xlen == 64
```

Comment:
The check for `__riscv_xlen` should not be part of the same condition, it can cause warnings on non-RISC-V platforms due to `__riscv_xlen` being used without being defined.

Reply:
I can change it, but it's already like this in `modules/utils/include/utils/system.h`

Reply:
Yeah, it's probably best to check `__riscv`, then inside that, check `__riscv_xlen` is defined as 64 or 32 or `#error` otherwise.